android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -361,7 +379,8 @@ private Bitmap loadArtwork(String url, boolean local) {
         Bitmap bitmap = null;
 
         try {
-            if(local) {
+            // If we are running the app in debug mode, the "local" image will be served from htt://localhost:8080, so we need to check for this case and load those images from URL
+            if(local && !url.startsWith("http")) {
```

Comment:
Are you suggesting that I copy your code over to work here?  I love the idea of react-native-track-player btw and will probably switch to that once it's ready for production.